i3pystatus/mail/imap.py

```diff
@@ -84,47 +41,56 @@ def init(self):
         if self.ssl:
             self.imap_class = IMAP4_SSL
 
-        self.conn = self.get_connection()
```

Comment:
Interesting, seems this was off for quite some time, since all other code uses self.connection, not self.conn